internal/services/containerapps/container_app_environment_storage_resource.go

```diff
@@ -95,6 +98,25 @@ func (r ContainerAppEnvironmentStorageResource) Arguments() map[string]*pluginsd
 			}, false),
 			Description: "The access mode to connect this storage to the Container App. Possible values include `ReadOnly` and `ReadWrite`.",
 		},
+
+		"azure_file_protocol": {
+			Type:     pluginsdk.TypeString,
+			Optional: true,
+			ForceNew: true,
+			Default:  "SMB",
+			ValidateFunc: validation.StringInSlice([]string{
+				"NFS",
+				"SMB",
+			}, false),
+		},
```

Comment:
I'm not sure we need this virtual property that doesn't represent anything in the API?

just make nfs_server and account_name exclusive and figure it out that way?

Reply:
Removed.